sdks/python/apache_beam/examples/ml-orchestration/kfp/components/train/src/train.py

```diff
@@ -0,0 +1,62 @@
+"""Dummy training function that loads a pretrained model from the torch hub and saves it."""
+
+import argparse
+from pathlib import Path
+import time
+
+import torch
+
+
+def parse_args():
+  """Parse ingestion arguments."""
+  parser = argparse.ArgumentParser()
+  parser.add_argument(
+    "--preprocessed-dataset-path", type=str,
+    help="Path to the preprocessed dataset.")
+  parser.add_argument(
+    "--trained-model-path", type=str,
+    help="Output path to the trained model.")
+  parser.add_argument(
+    "--base-artifact-path", type=str,
+    help="Base path to store pipeline artifacts.")
+  return parser.parse_args()
+
+
+def train_model(
```

Comment:
Can we add something like # [START train_component] and highlight this section in the orchestration.md docs?

Reply:
That makes sense, thanks for the explanation!. Let's leave it out for now.